src/assets/sass/protocol/base/elements/_links.scss

```diff
@@ -24,3 +24,22 @@
         color: $color-link-visited-hover;
     }
 }
+
+.mzp-c-cta-link {
+    font-weight: bold;
+
+    &:link,
+    &:visited {
+        border-bottom: 1px solid transparent;
+        color: $color-primary;
+        text-decoration: none;
+    }
+
+    &:hover,
+    &:active,
+    &:focus {
+        @include transition(border-bottom .1s ease-in-out);
```

Comment:
The nittiest of nitpicks: I think this could be border-bottom-color, though I doubt that makes any real difference. 

I've also been favoring milliseconds over decimals lately, based on something I saw in a presentation by someone... somewhere... the gist being "browsers think in milliseconds anyway, you might as well declare them explicitly and save them some work." So this could be `100ms` instead of `.1s`

Reply:
Updated